src/main/java/co/rsk/bitcoinj/core/Utils.java

```diff
@@ -85,6 +85,30 @@
         return bytes;        
     }
 
+    public static byte[] unsignedLongToByteArray(long number, int numBytes) {
+        if (number < 0) {
+            throw new VerificationException("Number needs to be positive");
+        }
+        validateNumberFitsInByteArray(number, numBytes);
+
+        byte[] result = new byte[numBytes];
+        for (int i = numBytes - 1; i >= 0; i--) {
+            result[i] = (byte)(number & 0xFF);
+            number >>= Byte.SIZE;
```

Comment:
Nitpick because this is theoretical at this point, but using `Byte.SIZE` makes this dependent on the platform and Bitcoin expects bytes of 8 bits. 

AFAIK every modern platform uses bytes of 8 bits, but...